integration_tests/test_ibc.py

```diff
@@ -74,3 +80,74 @@ def check_balance_change():
     assert_balance(cli2, ibc.ibc2.w3, escrow_address(port, channel)) == dst_amount
     assert_dynamic_fee(cli)
     assert_dup_events(cli)
+
+    ibc_denom_addr = ibc_denom_address(dst_denom)
+    w3 = ibc.ibc1.w3
+    erc20_contract = get_contract(w3, ibc_denom_addr, CONTRACTS["IERC20"])
+    assert erc20_contract.caller.decimals() == 0
```

Comment:
@yihuang we need [IERC20Metadata](https://github.com/cosmos/evm/blob/v0.3.0/precompiles/erc20/IERC20Metadata.sol#L27) right?